src/main/primary_entry/script/primary_entry.js

```diff
@@ -31,8 +35,10 @@ import "../images/red_dot.svg";
 import "../images/red_dot_t.svg";
 import "../images/red_dot_limited.svg";
 import "../images/orange_triangle.svg";
+import "../images/custom_pin.svg";
 
-
+import "../images/red_dot_expansion_permit.svg";
```

Comment:
We'll probably want to update these at some point to the pie-chart-like designs we discussed recently...